src/Ball1.jl

```diff
@@ -87,3 +89,51 @@ function σ(d::AbstractVector{N},
     res[imax] = sign(d[imax]) * B.radius
     return res
 end
+
+"""
+    ∈(x::AbstractVector{N}, B::Ball1{N})::Bool where {N<:Real}
+
+Check whether a given point is contained in a ball in the 1-norm.
+
+### Input
+
+- `x` -- point/vector
+- `B` -- ball in the 1-norm
+
+### Output
+
+`true` iff ``x ∈ B``.
+
+### Notes
+
+This implementation is worst-case optimized, i.e., it is optimistic and first
+computes (s. below) the whole sum before comparing to the radius.
+In applications where the point is typically far away from the ball, a fail-fast
+implementation with interleaved comparisons could be more efficient.
+
+### Algorithm
+
+Let ``B`` be an ``n``-dimensional ball in the 1-norm with radius ``r`` and let
+``c_i`` and ``x_i`` be the ball's center and the vector ``x`` in dimension
+``i``, respectively.
+Then ``x ∈ B`` iff ``∑_{i=1}^n |c_i - x_i| ≤ r``.
+
+### Examples
+
+```jldoctest
+julia> B = Ball1([1., 1.], 1.);
+
+julia> ∈([.5, -.5], B)
+false
+julia> ∈([.5, 1.5], B)
+true
+```
+"""
+function ∈(x::AbstractVector{N}, B::Ball1{N})::Bool where {N<:Real}
+    @assert length(x) == dim(B)
+    sum = zero(N)
+    for i in eachindex(x)
+        sum += abs(B.center[i] - x[i])
+    end
+    return sum <= B.radius
```

Comment:
i tend to prefer the more synthetic `return @. sum(abs(B.center - x)) <= B.radius` if possible. in julia we can have both with the same performance 👍 

the assertion is not "needed" since it is raised whenever dims do not match for the `-` operator, giving `ERROR: DimensionMismatch("dimensions must match")` (in other words i see no difference of having our assertion error rather than this DimensionMismatch error).

Reply:
`return @. sum(abs(B.center - x)) <= B.radius` does not work because the output is a Boolean array. When I wrap it in an `all(...)` block, the performance is poor.
```julia
julia> @time alternative([.5, 1.5], B)
  0.000023 seconds (9 allocations: 4.578 KiB)
julia> @time ∈([.5, 1.5], B)
  0.000011 seconds (5 allocations: 256 bytes)
```

Reply:
did i miss some parentheses? like `@.(sum(abs(B.center - x)))<= B.radius` 
it will have the same performance as the loops, no?

Reply:
Now I used `sum(@. abs(B.center - x)) <= B.radius`
```julia
julia> @time alternative([.5, 1.5], B)
  0.000010 seconds (6 allocations: 352 bytes)
```
Still not as good as the explicit form.

Reply:
> Still not as good as the explicit form.

at this scale we'll get some better comparison with `@btime` from `BenchmarkTools`

Reply:
I'm only looking at the memory.

Reply:
what does `@allocated`  say, and did you try declaring the variable `B` as `const`? recall that [measuring in the global scope can give misleading results](https://discourse.julialang.org/t/allocations-in-a-copy-construct/7323/2?u=mforets)

Reply:
I tried it only in the REPL. There I cannot declare as `const`.

Reply:
```julia
julia> @allocated ∈([.5, -.5], B)
96
julia> @allocated alternative([.5, -.5], B)
192
```